README.md

```diff
@@ -7,21 +7,25 @@
 Install [pnpm](https://pnpm.io/installation#using-npm) globally:
 
 ```
-npm i -g pnpm
+npm i -g pnpm@8
 pnpm i
 pnpm setup
 ```
 
+(Use pnpm v8, because Nx doesn't support v9.)
```

Comment:
Interesting. I wasn't aware of issues with v9 and we have been using it for a while. I will have to keep an eye on this.

Reply:
@irahopkinson pnpm v8 uses lockfile v6 and pnpm v9 uses lockfile v9. I think the issues I'm hitting might be related to the new lockfile format, but I'm not sure.